File: examples/create_and_delete_directory/create_and_delete_directory.php
<?php
$root_dir = './example_directoy';
$lang_dir = 'php';
$dirpath = implode('/', [$root_dir, $lang_dir]);
if( !is_dir($dirpath) ) {
  mkdir($dirpath, 0777, true);
}
if( is_dir($dirpath) ) {
  print "Succeeded in creation of ".$dirpath."\n";
  if(preg_match('/[a-zA-Z\d_\-]/', $root_dir)) {
    delTree($root_dir);
  }
  if( !is_dir($root_dir) ) {
    print "Succeeded in removal of ".$root_dir."\n";
  }
}

function delTree($dir) { 
  $files = array_diff(scandir($dir), array('.','..')); 
   foreach ($files as $file) { 
     (is_dir("$dir/$file")) ? delTree("$dir/$file") : unlink("$dir/$file"); 
   } 
   return rmdir($dir); 
 } 

File: examples/create_and_delete_directory/create_and_delete_directory.py
import os
import re
import shutil

root_dir = './example_directoy'
lang_dir = 'python'
dirpath = '/'.join([root_dir, lang_dir])
if not os.path.isdir(dirpath):
  os.makedirs(dirpath)

if os.path.isdir(dirpath):
  print("Succeeded in creation of " + dirpath)
  if re.search(r"[a-zA-Z\d_\-]", root_dir):
    shutil.rmtree(root_dir)
    if not os.path.isdir(root_dir):
      print("Succeeded in removal of " + root_dir)
